src/cmdline_sprite.c

```diff
@@ -2,6 +2,8 @@
 #include "cmdline.h"
 #include "drawing/drawing.h"
 #include "util/util.h"
+#include <math.h>
+#include "platform\platform.h"
```

Comment:
Use forward slash, I think this is why Travis is failing.
And if you can move <math.h> above the local header includes.